composition-js/src/__tests__/compose.test.ts

```diff
@@ -1,55 +1,26 @@
 import {
   asFed2SubgraphDocument,
   assert,
-  buildSchema,
   buildSubgraph,
-  extractSubgraphsFromSupergraph,
   FEDERATION2_LINK_WTH_FULL_IMPORTS,
   inaccessibleIdentity,
   InputObjectType,
   isObjectType,
   ObjectType,
   printSchema,
   printType,
-  Schema,
-  ServiceDefinition,
-  Subgraphs
 } from '@apollo/federation-internals';
-import { CompositionResult, composeServices, CompositionSuccess } from '../compose';
+import { CompositionResult, composeServices } from '../compose';
 import gql from 'graphql-tag';
 import './matchers';
 import { print } from 'graphql';
```

Comment:
Note to reviewers: I noticed that `compose.test.ts` tests were getting run multiple times because the helper functions were being imported from other files, so I split those out into a separate file.